proposals/memory-specializations/intro.tex

```diff
@@ -0,0 +1,72 @@
+\rSec0[intro]{General}
+\rSec1[intro.scope]{Scope}
+
+\pnum
+This proposal describes functions compatible with the Ranges Technical
+Specification~(\ref{intro.refs}) that shall not be ratified prior to the
+publication of the TS. These extensions change and add to the existing
+memory specialisation library facilities found in N4606~(\ref{intro.refs}).
+
+\rSec1[intro.refs]{References}
+This document requires some familiarity with the following documents.
+
+\begin{itemize}
+\item JTC1/SC22/WG21 N4606,   \doccite{Working Draft, Standard for Programming Language \Cpp}
```

Comment:
Better to base this document on C++14, at least until the Ranges TS is retargeted to C++17.